pkg/assembler/backends/backends.go

```diff
@@ -53,6 +53,7 @@ type Backend interface {
 	VulnerabilityMetadata(ctx context.Context, vulnerabilityMetadataSpec *model.VulnerabilityMetadataSpec) ([]*model.VulnerabilityMetadata, error)
 
 	// Mutations for software trees (read-write queries)
+<<<<<<< Updated upstream
```

Comment:
This looks like a merge conflict marker

Reply:
I'll take care of this. Thank you for pointing out!